src/features/notifications/NotificationShow.tsx

```diff
@@ -51,7 +51,7 @@ const NotificationShow = ({ notification, onClose }: Props) => {
   )
 
   const bodyStyle = useMemo(
-    () => ({ maxHeight: Platform.OS === 'ios' ? 420 : 320 }),
+    () => ({ maxHeight: Platform.OS === 'ios' ? 450 : 350 }),
```

Comment:
This is totally fine as is, but I recently realized there's also a `Platform.select` function that I hadn't been utilizing.

So this could be...
```ts
  const bodyStyle = useMemo(
    () => ({ maxHeight: Platform.select({ ios: 450, android: 350 }) }),
    [],
  )
```